src/collective/lazysizes/transform.py

```diff
@@ -154,16 +156,17 @@ def transformUnicode(self, result, encoding):
         return
 
     def transformIterable(self, result, encoding):
-        if not api.user.is_anonymous():
+        registry = getUtility(IRegistry)
+        settings = registry.forInterface(ILazySizesSettings)
+
+        if not (api.user.is_anonymous() or settings.lazyload_authenticated):
```

Comment:
It took me quite a while to understand this, wouldn't make sense to rewrite it as:

```if settings.lazyload_authenticated or not api.user.is_anonymous()```

At least to me it makes it a bit more clear